bundle/get_started.md

```diff
@@ -0,0 +1,414 @@
+
+# Get started to MONAI bundle
+
+A MONAI bundle usually includes the stored weights of a model, TorchScript model, JSON files which include configs and metadata about the model, information for constructing training, inference, and post-processing transform sequences, plain-text description, legal information, and other data the model creator wishes to include.
+
+For more information about MONAI bundle, please read the description: https://docs.monai.io/en/latest/bundle_intro.html.
+
+This is a step-by-step tutorial to help get started to develop a bundle package, which contains a config file to construct the training pipeline and also has a `metadata.json` file to define the metadata information.
+
+Mainly contains the below sections:
+- Define a training config with `JSON` or `YAML` format.
+- Execute training based on bundle scripts and configs.
+- Execute other scripts for bundle functionalities.
+- Hybrid programming with config and python code.
+
+You can find the usage examples of MONAI bundle key features and syntax in this tutorial, like:
+- Instantiate a python object from a dictionary config with `_target_` indicating class or function name or module path.
+- Execute python expression from a string config with the `$` syntax.
+- Refer to other python object with the `@` syntax.
+- Macro text replacement with the `%` syntax to simplify the config content.
+- Leverage the `_disabled_` syntax to tune or debug different components.
+- Override config content at runtime.
+- Hybrid programming with config and python code.
+
+## Download dataset
+
+Downloads and extracts the dataset for this example.
+The dataset comes from http://medicaldecathlon.com/.
+Here specify a directory with the `MONAI_DATA_DIRECTORY` environment variable to save downloaded dataset and outputs.
+
+```python
+import os
+from monai.apps import download_and_extract
+
+resource = "https://msd-for-monai.s3-us-west-2.amazonaws.com/Task09_Spleen.tar"
+md5 = "410d4a301da4e5b2f6f86ec3ddba524e"
+
+root_dir = os.environ.get("MONAI_DATA_DIRECTORY")
```

Comment:
I would suggest to use `tempfile` to make temporary folder if the users do not have the env var MONAI_DATA_DIRECTORY
```
directory = os.environ.get("MONAI_DATA_DIRECTORY")
root_dir = tempfile.mkdtemp() if directory is None else directory
print(root_dir)
```

Reply:
Sure, updated in the latest commit.

Thanks.